spec/shippo/api/model/track_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+CARRIER = 'usps'
+TRACKING_NO = '9205590164917312751089'
+
+RSpec.describe 'Shippo::API::Track' do
+  configure_vcr('spec/fixtures/track')
+
+  let(:params) { { 'carrier': CARRIER,
+                   'tracking_number': TRACKING_NO }}
+
+  let(:invalid_params) { { 'carrier': "INVALID_CARRIER",
+                   'tracking_number': TRACKING_NO }}
+
+  describe '#retrieve' do
+    it 'should properly return a Track object' do
+      VCR.use_cassette("test_retrieve") do
+        track = Shippo::Track::get(TRACKING_NO, CARRIER)
+        expect(track).to be_kind_of(Shippo::Track)
+        expect(track.tracking_number).to be == TRACKING_NO
+        expect(track.tracking_history).not_to be == nil
+      end
+    end
+  end
+
+  describe '#invalid_retrieve' do
+    it 'should raise an exception' do
+      VCR.use_cassette("test_invalid_retrieve") do
+        expect {
+          Shippo::Track::get(TRACKING_NO, "INVALID_CARRIER")
+        }.to raise_error(Shippo::Exceptions::Error)
+      end
+    end
+  end
+
+  describe '#invalid_retrieve_no_carrier' do
+    it 'should raise an exception' do
+      VCR.use_cassette("test_invalid_retrieve_no_carrier") do
+        expect {
+          Shippo::Track::get(TRACKING_NO)
+        }.to raise_error(Shippo::Exceptions::Error)
```

Comment:
This test is removed now since the method signature requires that a carrier is provided